.travis.yml

```diff
@@ -9,6 +8,7 @@ cache:
 
 install:
   - pip install .
+  - pip install -r requirements.txt
```

Comment:
This should not be necessary. The `pip install .` above runs the `setup.py` which reads and parses `requirements.txt` and passes the entries to `setup(..., install_requires=requirements)`. Make sense?

Reply:
👍